.goreleaser.yaml

```diff
@@ -0,0 +1,43 @@
+# For more options see: https://goreleaser.com/customization
+
+before:
+  hooks:
+  - go mod tidy
+  - go mod vendor
+
+project_name: log-cache
+
+builds:
+- env:
+  - CGO_ENABLED=0
+  goos:
+  - linux
+  - windows
+  - darwin
+
+archives:
+- format: tar.gz
```

Comment:
Need to figure out how to add LDFLAGs to the build so that we can specify the version for the built binaries.